veascan-web/src/consts/bridges.ts

```diff
@@ -1,7 +1,7 @@
 import VeaInboxArbitrumSepolia from "@kleros/vea-contracts/deployments/arbitrumSepolia/VeaInboxArbToEthDevnet.json";
 import VeaOutboxSepolia from "@kleros/vea-contracts/deployments/sepolia/VeaOutboxArbToEthDevnet.json";
-import VeaInboxArbitrumSepoliaChiado from "@kleros/vea-contracts/deployments/arbitrumSepolia/VeaInboxArbToGnosisDevnet.json";
-import VeaOutboxArbitrumSepoliaChiado from "@kleros/vea-contracts/deployments/chiado/VeaOutboxArbToGnosisDevnet.json";
+// import VeaInboxArbitrumSepoliaChiado from "@kleros/vea-contracts/deployments/arbitrumSepolia/VeaInboxArbToGnosisDevnet.json";
```

Comment:
Is there any intentional reason to leave this commented code lines or did it just slip?

Reply:
Commented to get veascan running for a single route for now, will uncomment with the updated deployments for the other route (Arbitrum Sepolia to Chiado).

Reply:
Thanks for the clarification, @mani99brar! I'll resolve the comment